builder-base/Makefile

```diff
@@ -25,6 +27,7 @@ local-images:
 		--local dockerfile=./ \
 		--local context=. \
 		--output type=tar,dest=/tmp/builder-base/output.tar
+	./update_base_image.sh --dry-run
```

Comment:
Any reason we aren't cd'ing into the pr-bot dir like we do for the images target below?

Reply:
So all the code in the eks-distro-pr-bot/REPO is whatever was merged into the main aws/REPO. So just wanted to test against main in the presubmit, so that we dont have to think about whether the code is updated in the bot's fork or not. The job does the fetch and rebase in the create_pr script for each of the repos which need automated PR's for, but it doesn't fetch from upstream before calling scripts, so if it can't find the create_pr script, it will fail. I can have it fetch and rebase in the Makefile then maybe?

Reply:
I feel like the source of these scripts should be in the main aws/repo, and inside the script it should cd into the necessary directories, not within this makefile itself